dashboard/static/styles/main.styl

```diff
@@ -26,6 +26,11 @@ section {
     background-position: top center;
 }
 
+.top-bar-title h1 {
+    font-size: 1rem;
+    font-weight: normal;
```

Comment:
Hm, maybe we're seeing different things. It looks like Foundation sets the default `font-weight` to 400 which is equivalent to normal? Removing the `font-weight: normal;` rule here doesn't change the appearance of the text.